spec/requests/user_authentication_spec.rb

```diff
@@ -0,0 +1,66 @@
+require "rails_helper"
+
+RSpec.describe "User Authentication", type: :request do
+
+  context "when trying to create a user" do
+    it "returns an auth token" do
+      VCR.use_cassette("synopsis") do
+        token = "EAAH6msXVZCB0BAHqwjgc829cZAZAg6Ymua77S9rkmkLDvr701mavIxjZBsGnihhJ5roGcY2vTCWb8nmi78LNk4NmNkgAuSK660oHmWkKNdXRAIIPy7qDEugpt78VvGvzzpFeOxc2t9xsZBN4viopXh2utRfIZB1WEZD"
```

Comment:
Since we already have a VCR record of this request with a valid token we can avoid exposing it.